string locid
---
string result
---
string feedback
